libsrc/osic1p/osic1p.inc

```diff
@@ -0,0 +1,5 @@
+
+SCRNBASE        := $D000    ; Base of video RAM
+VIDEORAMSIZE    := $0400    ; Size of C1P video RAM (1 kB)

```

Comment:
The ca65 assembler has two types of assignment operators: label (":=") and equate ("=").  Ca65 doesn't care which one we use; but, they do affect [the information](https://github.com/cc65/wiki/wiki/Debug-info-data#symbols) in [a debug file](https://github.com/cc65/wiki/wiki#debug-information).

`VIDEORAMSIZE` isn't an address; therefore, you should use the equate operator to create it.